File: erp-frontend/src/api/material/order/model/queryMaterialOrderVo.ts
import { SortPageVo } from '@/api/model/sortPageVo';

export interface QueryMaterialOrderVo extends SortPageVo {
  /**
   * 发料单号
   */
  code?: string;

  /**
   * 仓库ID
   */
  scId?: string;

  /**
   * 合同编号
   */
  contractCode?: string;

  /**
   * 发料申请单ID
   */
  materialApplyId?: string;

  /**
   * 是否已完成出库
   */
  isOutFinish?: boolean;

  /**
   * 操作人
   */
  createBy?: string;

  /**
   * 操作起始时间
   */
  createTimeStart?: string;

  /**
   * 操作截止时间
   */
  createTimeEnd?: string;
}

File: erp-backend/xingyun-shkb/src/main/java/com/lframework/xingyun/shkb/vo/material/QueryMaterialOrderVo.java
package com.lframework.xingyun.shkb.vo.material;

import com.lframework.starter.web.core.vo.PageVo;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

import java.time.LocalDateTime;

/**
 * 发料单查询Vo
 *
 * @author kison
 */
@Data
public class QueryMaterialOrderVo extends PageVo {

    private static final long serialVersionUID = 1L;

    /**
     * 单号
     */
    @ApiModelProperty("单号")
    private String code;

    /**
     * 仓库ID
     */
    @ApiModelProperty("仓库ID")
    private String scId;

    /**
     * 合同编号
     */
    @ApiModelProperty("合同编号")
    private String contractCode;

    /**
     * 创建人
     */
    @ApiModelProperty("创建人")
    private String createBy;

    /**
     * 创建时间-起
     */
    @ApiModelProperty("创建时间-起")
    private LocalDateTime createTimeStart;

    /**
     * 创建时间-止
     */
    @ApiModelProperty("创建时间-止")
    private LocalDateTime createTimeEnd;

    /**
     * 发料申请单ID
     */
    @ApiModelProperty("发料申请单ID")
    private String materialApplyId;

    /**
     * 是否已完成出库
     */
    @ApiModelProperty("是否已完成出库")
    private Boolean isOutFinish;
}
